projects/plugins/jetpack/modules/sharedaddy/sharing.php

```diff
@@ -455,7 +455,7 @@ public function management_page() {
 					</td>
 					<td class="services">
 						<h2 <?php echo ( count( $enabled['all'] ) > 0 ) ? ' style="display: none"' : ''; ?>><?php esc_html_e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>
-						<div class="sharedaddy sd-sharing-enabled">
+						<div class="sharedaddy sd-sharing-enabled <?php if ( $global['button_style'] === 'official' ) { echo ''; } ?>">
```

Comment:
Is this necessary since we're not actually outputting anything new?

Reply:
Haha. How embarrassing! I originally had this hardcoded, but then figured out that dynamically setting it with JS was the best way forward, but forgot to remove this line. Thanks for catching this!